foreign_cc/private/framework/toolchains/default_commands.bzl

```diff
@@ -93,15 +94,15 @@ local target="$2"
 mkdir -p "$target"
 if [[ -f "$1" ]]; then
   ln -s -f -t "$target" "$1"
-elif [[ -L "$1" ]]; then
-  local actual=$(readlink "$1")
-  ##symlink_to_dir## "$actual" "$target"
+elif [[ -L "$1" && ! -d "$1" ]]; then
+  cp "$1" "$2"
```

Comment:
Should this be `cp -a`?

Reply:
This is exactly what the linux commands do.
https://github.com/bazelbuild/rules_foreign_cc/blob/98c88de5e0f57dd2a896dd595469ccd6bf97ceab/foreign_cc/private/shell_toolchain/toolchains/impl/linux_commands.bzl#L97

should that be `cp -a`?

Reply:
the default (stale linux) toolchain was instead deleted.